test/fixtures/spawn_closed_stdio.py

```diff
@@ -0,0 +1,9 @@
+import os
+import sys
+import subprocess
+os.close(0)
+os.close(1)
+os.close(2)
+cmd = sys.argv[1] + ' -e "process.stdin; process.stdout; process.stderr;"'

```

Comment:
If you do use -e, write this as `cmd = sys.argv[1:2] + ['-e', 'process.stdin; ...']` so it gets escaped properly.